crates/multi_buffer/src/multi_buffer.rs

```diff
@@ -87,6 +92,14 @@ pub struct MultiBuffer {
     /// The writing capability of the multi-buffer.
     capability: Capability,
     buffer_changed_since_sync: Rc<Cell<bool>>,
+    follower: Option<Entity<MultiBuffer>>,
+    filter_mode: Option<MultiBufferFilterMode>,
```

Comment:
Can we add docs to this, multi buffer is already so complicated and hard to grasp as is 😅